core/include/thread.h

```diff
@@ -133,6 +133,10 @@
 extern "C" {
 #endif
 
+#if defined(DEVELHELP) && !defined(CONFIG_THREAD_NAMES)
+#define CONFIG_THREAD_NAMES
```

Comment:
The static tests are asking for some Doxygen documentation here

Reply:
Fixed (directly amended)